### PR TITLE
Adding information in system test template about testing in EC2 

### PR DESCRIPTION
**Pull request by pro-akim**
|Related issue|
|-|
|wazuh-qa#4527|

## Description

Adding information to the system test template about how to run system tests in EC2.

It contains:

- EC2 requirements per environment
- Relevant notes
- Instructions to create the environment in the EC2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 1]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+| Environment                  | EC2                                       |
+|------------------------------|-------------------------------------------|
+|Basic_cluster                 |Ubuntu 22.04.2 LTS `C5`.`XLarge` `15GB` HD |
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large `60GB` HD      |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        | 
+| Basic_environment            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
```
Comment (rauldpm) [inline .github/ISSUE_TEMPLATE/system-tests.md:33]: AWS does not provide HD but HDD, also remove the extra space in line 33 between `|` and `Basic`

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 2]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+
+
+These requirements should be requested from the @cicd-team.
```
Comment (rauldpm) [inline .github/ISSUE_TEMPLATE/system-tests.md:36]: I do not think we should hardcode the Slack team. What if the team name changes? Set the GitHub team instead: @wazuh/cicd or refer to the IT team

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 3]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+For its execution, the installation of various packages is required, which will be detailed below.
+
+<details><summary>Steps</summary>
+
+```
+### Updating dependencies
+sudo apt update
+sudo apt upgrade
+
+### Installing git and cloning wazuh-qa project
```
Comment (rauldpm) [inline .github/ISSUE_TEMPLATE/system-tests.md:48]: Change `project` to `repository`

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 4]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+
+### Installing requirements and additional libraries
+python3 -m pip install -r /home/ubuntu/wazuh-qa/requirements.txt
+sudo apt install python3-pip
+sudo apt install python3.10-venv
+cd /home/ubuntu/wazuh-qa/deps/wazuh_testing
+python3 -m pip install .
+```
+
+For further information: check /wazuh-qa/tests/system/README.md
```
Comment (rauldpm) [inline .github/ISSUE_TEMPLATE/system-tests.md:84]: Set a complete URL

**Review by rauldpm — CHANGES_REQUESTED** [review]
Minor changes

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 4 above)
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 3 above)
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 2 above)
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 1 above)
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 5]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large 60GB SSD       |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       | 
+|Basic_environment             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+
+
+These requirements should be requested from the @wazuh/cicd.
```
Comment (verdx) [inline .github/ISSUE_TEMPLATE/system-tests.md:36]: "the @wazuh/cicd team"

**Review by verdx — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done in https://github.com/wazuh/wazuh/pull/19080/commits/b60b0df62e6630ccec7bb6d3a1b1696381955507

**Review by verdx — DISMISSED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 6]
```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
...
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large 60GB SSD       |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       | 
+|Basic_environment             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+
+
+These requirements should be requested from the @wazuh/cicd team.
```
Comment (juliamagan) [inline .github/ISSUE_TEMPLATE/system-tests.md:36]: Please, add the `internal-devel-request` link

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/system-tests.md
[the remark this answers is not in the document]
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md:1]: Done https://github.com/wazuh/wazuh/pull/19080/commits/835047378f9657b84edc30e4325fecc1bfe3cfc6

File: .github/ISSUE_TEMPLATE/system-tests.md (hunk 6 above)
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done https://github.com/wazuh/wazuh/pull/19080/commits/835047378f9657b84edc30e4325fecc1bfe3cfc6

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/system-tests.md [hunk 7]
```diff
@@ -18,6 +18,25 @@ The following issue aims to run all `system tests` for the current release candi
...
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       | 
+|Basic_environment             |Ubuntu 22.04.2 LTS T3.Large 30GB SSD       |
+
+
+These requirements should be requested to the @wazuh/cicd team via https://github.com/wazuh/internal-devel-requests.
+
+For further information: check https://github.com/wazuh/wazuh-qa/tree/master/tests/system/README.md
```
[the remark this answers is not in the document]
Comment (pro-akim) [reply .github/ISSUE_TEMPLATE/system-tests.md]: Done https://github.com/wazuh/wazuh/pull/19080/commits/44b5d547646e9a25865964805182c6ee9c20e370

**Pull request merged**